app/templates/home.twig

```diff
@@ -28,12 +28,12 @@
                         <a class="btn btn-primary btn-lg" href="{{ '/integration'|completeUrl }}">Integrate Matomo</a>
                     </p>
                 </div>
-                <div class="col-sm-6">
+                <div class="category">
                     <h2>
-                        <i class="fa fa-code"></i> Develop
+                        <i class="fa fa-code"></i></br>Develop
```

Comment:
@tsteur Should be `<br>` or `<br/>`

Reply:
That's fixed now. Thanks for catching it 👍